utils/scripts/reportportal/rp_uploader.py

```diff
@@ -0,0 +1,102 @@
+import os
+import argparse
+import sys
+import json
+
+dir_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(dir_path+"/../")
+
+from util import execute_command
+from logger import log
+
+"""
+Class for Report portal operations
+"""
+
+
+class reportPortalOperations():
```

Comment:
i think uppercase initial class name should be there

Reply:
Comments Incorporated